packages/client/src/subscribe/OrderMessages.ts

```diff
@@ -43,8 +43,8 @@ export class OrderMessages<T> {
         this.onOrdered = this.onOrdered.bind(this)
         this.onGap = this.onGap.bind(this)
         this.maybeClose = this.maybeClose.bind(this)
-        const { gapFillTimeout, retryResendAfter, maxGapRequests, orderMessages = true, gapFill = true } = this.config
-        this.enabled = !!(gapFill && maxGapRequests)
+        const { gapFillTimeout, retryResendAfter, maxGapRequests, orderMessages, gapFill } = this.config
```

Comment:
What about the default values before (i.e. `orderMessages = true` and `gapFill = true`)? Is this functionally equivalent?

Reply:
Not a functionality change, the fallbacks were never used. Same default values in https://github.com/streamr-dev/network/blob/6753475f770de8dda94cdea7252603aad8c187b5/packages/client/src/Config.ts#L137